src/components/OutputsWrapper.js

```diff
@@ -47,12 +47,12 @@ class OutputsWrapper extends React.Component {
         <input type="text" ref={this.address} placeholder="Address" className="form-control output-address col-4" />
         <input type="number" ref={this.value} step={hathorLib.helpers.prettyValue(1)} min={hathorLib.helpers.prettyValue(1)} placeholder={hathorLib.helpers.prettyValue(0)} className="form-control output-value col-2" />
         <div className="form-check mr-3 d-flex flex-column justify-content-center">
-          <input className="form-check-input mt-0 has-timelock" ref={this.timelockCheckbox} type="checkbox" onChange={this.handleCheckboxTimelockChange} id={this.uniqueID}/>
+          <input className="form-check-input mt-0 has-timelock" title={hathorLib.wallet.isHardwareWallet() ? "This feature is disabled for hardware wallet" : "Timelock"} disabled={hathorLib.wallet.isHardwareWallet() ? true : null} ref={this.timelockCheckbox} type="checkbox" onChange={this.handleCheckboxTimelockChange} id={this.uniqueID}/>
```

Comment:
Would you refactor and break it into multiple lines? I think it is easier to read and review.

Reply:
Done